doc/api/addons.md

```diff
@@ -1,3 +1,6 @@
+<!--
+Run `node tools/doc/addon-verify.js` when you change examples in this document.
```

Comment:
Is there a way we could validate this? Maybe a test that contains the shasum of everything inside a `js` code block in this file, that you have to update when it changes.

Reply:
I've added a commit that adds a `--check` flag to addon-verify.js and calls that from `make test`.